invenio_app_ils/ui/src/invenio_app_ils/common/api/documents/document.js

```diff
@@ -45,8 +45,8 @@ const queryBuilder = () => {
 
 const list = query => {
   return http.get(`${documentURL}?q=${query}`).then(response => {
-    const totalHits = response.data.hits.total;
```

Comment:
`totalHits` is undefined on line 52 now so maybe just keep the variable definition here?